File: src/apps/Andriod/app/src/main/java/com/dkellycollins/triggerfinger/data/daos/ITouchPositionDao.java
package com.dkellycollins.triggerfinger.data.daos;

import com.dkellycollins.triggerfinger.data.model.IVector;

/**
 * Provides the ability to retrieve touch data.
 */
public interface ITouchPositionDao {
    IVector getLastPosition();
    boolean isTouching();
}

File: src/apps/Android-Kotlin/app/src/main/kotlin/com/dkellycollins/triggerfinger/data/daos/ITouchPositionDao.kt
package com.dkellycollins.triggerfinger.data.daos

import com.dkellycollins.triggerfinger.data.model.IVector

/**
 * Provides the ability to retrieve touch data.
 */
interface ITouchPositionDao {
    val lastPosition: IVector
    val isTouching: Boolean
}
